### PR TITLE
add wrtds, refine loadest forecast tasks

### PR DESCRIPTION
**Pull request by aappling-usgs**
* add `forecast_wrtds`
* refine `forecast_loadest` to produce same core columns of forecast output as `forecast_wrtds`
* modify `prep_inputs` to facilitate preliminary testing of forecast_wrtds and forecast_loadest
* modify remake files to run forecast_wrtds and forecast_loadest

all this will need to be debugged further when we have data

### CODE REVIEW

File: 3_forecast/src/apply_loadest.R [hunk 1]
```diff
@@ -1,30 +1,40 @@
...
+      flow="Q",
+      dates="Date",
+      conc.units=units_lookup[eList$INFO$param.units],
+      flow.units='cms',
+      station=eList$INFO$site.no)
+
+    # Generate load forecasts
+    est_data <- eList$Daily %>%
+      select(Date, Q) %>%
+      filter(Date >= eList$INFO$ref_date)
```
Comment (jzwart) [inline 3_forecast/src/apply_loadest.R]: I think this is in agreement issue #34

File: 3_forecast/src/apply_loadest.R [hunk 2]
```diff
@@ -1,30 +1,40 @@
...
-    flow="Q",
-    dates="Date",
-    conc.units=units_lookup[eList$INFO$param.units],
-    flow.units='cms',
-    station=eList$INFO$site.no)
+  sys_time <- system.time({
+    # Fit the 7-parameter model
+    fit_data <- eList$Sample %>%
+      select(Date, ConcLow, ConcHigh, Q) %>%
+      filter(Date < eList$INFO$ref_date)
```
Comment (jzwart) [inline 3_forecast/src/apply_loadest.R]: maybe it's just me, but I've been having trouble with `filter()` - `stats` package overrides `dplyr`.  I suggest `dplyr::filter()`

File: 3_forecast/src/apply_loadest.R [hunk 3]
```diff
@@ -1,30 +1,40 @@
...
-  preds <- predLoad(
-    fit, newdata=est_data, by='day'
-  ) %>%
-    mutate(site = eList$INFO$site.no)
+  # Write the full model output as an augmented eList
+  aList <- c(eList, list(fit_data=fit_data, est_data=est_data, fit=fit, preds=preds))
+  saveRDS(aList, log_rds)
 
   # Write the forecasts to file
   saveRDS(preds, output_rds)
```
Comment (jzwart) [inline 3_forecast/src/apply_loadest.R]: is this still needed if `preds` is in `aList` ?

File: 3_forecast/src/apply_loadest.R (hunk 2 above)
Comment (aappling-usgs) [reply 3_forecast/src/apply_loadest.R]: sounds good, i'll add `dplyr::`

File: 3_forecast/src/apply_loadest.R (hunk 3 above)
Comment (aappling-usgs) [reply 3_forecast/src/apply_loadest.R]: hmm...yeah, maybe not. i could just write the full list every time. will push changes.

**Comment by aappling-usgs** [comment]
ok, finished pushing suggested changes

**Pull request merged**